tests/test_pgo_benchmark.py

```diff
@@ -0,0 +1,53 @@
+import pytest
+import torch
+from omegaconf import OmegaConf
+
+import examples.pose_graph.pose_graph_synthetic as pgo
+
+
+@pytest.fixture
+def default_cfg():
+    cfg = OmegaConf.load("examples/configs/pose_graph/pose_graph_synthetic.yaml")
+    cfg.outer_optim.num_epochs = 1
+    cfg.outer_optim.max_num_batches = 4
+    cfg.batch_size = 16
+    cfg.num_poses = 64
+    cfg.profile = False
+    cfg.savemat = False
+    cfg.inner_optim.optimizer_kwargs.verbose = False
+    return cfg
+
+
+@pytest.mark.parametrize(
+    "linear_solver_cls",
+    ["CholeskyDenseSolver", "LUCudaSparseSolver", "CholmodSparseSolver"],
```

Comment:
Do we need to add `BaSpaCho`?

Reply:
I think I was going to add another test just for baspacho (to mark `run_if_baspacho`), but forgot to do it. I'll add, thanks.

Reply:
Got it. Feel free to merge.